js/account-popover.jquery.js

```diff
@@ -52,11 +67,20 @@
       wireEventHandlers: function wireEventHandlers() {
         var $trigger = this.$trigger;
 
+        if (this.options.name) {
```

Comment:
The badge HTML itself (should probably be an option?), or the insertion of the name (AFAIK it's always been in `wireEventHandlers`)?